SIL.Scripture.Tests/TestScrVers.cs

```diff
@@ -2,7 +2,7 @@
 #region // Copyright (c) 2014, SIL International.
 // <copyright from='2013' to='2014' company='SIL International'>
 //		Copyright (c) 2014, SIL International.   
-//    
+//	
```

Comment:
I wouldn't say it was "hard work" as it was just a find/replace 4 spaces with a single tab. :)